ProjectExample/Final_Project_Example.Rmd

```diff
@@ -117,8 +117,34 @@ ggplot(data = top5,
            y = Happiness_score )) +
     geom_point()
 ```
+I felt that this plot could be improved. So first I used I made `Country_name` a factor and then reordered it by the `Happiness_score` variable. 
+
+```{r}
+
+top5 <- top5 %>% mutate(Country_name = as_factor(Country_name))
```

Comment:
```suggestion
top5 <- top5 %>% mutate(Country_name = as_factor(Country_name), 
    Country_name = fct_reorder(Country_name, Happiness_score))
```
You can combine these in one step.

Reply:
yeah I did this to make it easier to understand - but indeed I think they can handle it so I will change it